source/release-notes/1.8.txt

```diff
@@ -21,9 +21,83 @@ Release Notes for mongosync 1.8
 
 **Upcoming**
 
-New Features:
+Updates to ``mongosync`` ``/progress`` API endpoint
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+- Returns a top-level boolean ``success`` field.
+
+- Returns the integer ``totalEventsApplied`` field, which approximates 
+  the number of change events applied to a destination cluster by an 
+  instance of ``mongosync``.
+
+- Reports per-partition progress tracking with the following changes:
+
+  - Calls to ``/progress`` before ``mongosync`` reaches the collection 
+    copy phase return 0 for ``estimatedCopiedBytes`` and 1 for 
+    ``estimatedTotalBytes``.
+
+  - ``estimatedTotalBytes`` may change throughout the collection copy 
+    phase if documents are inserted or deleted on the source cluster.
+
+  - ``estimatedCopiedBytes`` is never be greater than 
+    ``estimatedTotalBytes``. Progress reaches 100% at the end of 
+    the collection copy phase 
+    (``estimatedCopiedBytes`` = ``estimatedTotalBytes``).
+
+  - When performing a live upgrade from an earlier version to 1.8.0 or 
+    higher, the collection copy data starts over from 
+    ``[0 bytes copied / 1 bytes total]``. After a live upgrade, 
+    ``/progress`` only reports the progress of data copied after the 
+    upgrade completed.
+
+See the :ref:`c2c-api-progress` documentation for more details.
+
+Other Notes
+~~~~~~~~~~~
+
+Optimizations:
+
+- Fixes and enhancements to significantly reduce the time needed to create 
+  partitions and adds logging that tracks the partition creation process.
+
+Other Changes:
+
+- ``mongosync`` now performs a round robin assignment of the primary 
+  shard for each database if the destination is a sharded cluster.
+
+- ``mongosync`` now errors if it cannot create an index after 10 
+  attempts. Previously, the system would attempt to create an index 
+  indefinitely.
+
+- ``mongosync`` now sends hostnames as telemetry.
+
+- Improved performance of the initialization process by eliminating 
```

Comment:
Can we also update the bullet here that conflicts with this statement?

The last bullet in this [section](https://www.mongodb.com/docs/cluster-to-cluster-sync/current/reference/api/start/#required-indexes)

Reply:
Updated in DOCSP-41853 ([direct link to file diff for /start](https://github.com/mongodb/docs-cluster-to-cluster-sync/pull/362/files#diff-c8f234f7efcdc563e411e7b8709a62b79edece8cd33baa84eaa370453faa3acd))